device-types/Ubiquiti/U6-Enterprise.yaml

```diff
@@ -1,17 +1,23 @@
 ---
 manufacturer: Ubiquiti
-model: UniFi U6 Access Point Enterprise
+model: U6 Access Point Enterprise
```

Comment:
According to the datasheet this is a `Access Point U6 Enterprise`

Reply:
```suggestion
model: Access Point U6 Enterprise
```